src/components/DatasetSettings.vue

```diff
@@ -93,6 +93,16 @@
                 </p>
             </div>
 
+            <div>
+                <label class="flex mt-2 items-center">
+                    <input v-model="context.userClassificationFilters" class="accent-brand-500 mr-3 h-5 w-5" type="checkbox">
+                    Hide products not available for the user's <span class="font-medium px-1">Country</span> or <span class="font-medium px-1">Channel</span> classification
```

Comment:
```suggestion
                    Hide products not available for the user's <span class="font-medium px-1">country</span> or <span class="font-medium px-1">channel</span> classification
```

You used `<code>` for utm but here you use bold, should we not use the same stategy for keys here?

Reply:
Makes sense!